docs/api/catalog-resource.md

```diff
@@ -244,6 +245,15 @@ time before the catalog item's commit timestamp.
 The `packageHashAlgorithm` is a string defined by the server implementation representing the hashing algorithm used to
 produce the `packageHash`. nuget.org always used the `packageHashAlgorithm` value of `SHA512`.
 
+The `packageTypes` property will only be present if a package type was specified by the author. If it is present, it will always have at least one (1) entry. Each item in the `packageTypes` array is a JSON object with the following properties:
+
+Name      | Type    | Required | Notes
+--------- | ------- | -------- | -----
+@id       | string  | yes      | The URL to fetch the package type item
+@type     | string  | yes      | Type of package type. Currently will always be "PackageType"
```

Comment:
Let's leave `@id` and `@type` undocumented. They are useless and ideally we could drop them with no semantic impact.